README.md

```diff
@@ -9,7 +9,7 @@ A Dockerfile for Sonatype Nexus Repository Manager 3, based on CentOS.
 To run, binding the exposed port 8081 to the host.
 
```

Comment:
To remove any confusion, just change it to something like "Sonatype Nexus3 Docker" as I can't tell if the intent is to promote the github repo name, the docker name or whatever else